### PR TITLE
React Router Addition

### PR DESCRIPTION
**Pull request by AKeeganDev**
This pull request features the addition of React Router. My app can now be navigated through a top Navbar that holds code to transfer between the different "pages" of the app. The pages now load dynamically onto the page. There are 3 pages to choose from, the Home, Calculator, and Quote page.

### CODE REVIEW

**Review by Kingstalux — CHANGES_REQUESTED** [review]
Hi @AKeeganDev, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

149637855-7722d04c-b45d-4185-925f-7de047106461

## Highlights 👏 
✅ App works as expected 🔥 
✅ No linter errors 👍 
✅ Good commit message 👏 


## Required Changes ♻️
- [ ] Kindly edit your README file using this [Microverse template](https://github.com/microverseinc/readme-template) and provide all the necessary information a user would need to set up and run your project locally on their machine. This will make your repo more professional 💯 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 1]
```diff
@@ -1,3 +1,6 @@
+# Live Demo
+https://akeegandev.github.io/math_magicians/
+
 # Getting Started with Create React App
```
Comment (arslanbisharat) [inline README.md]: The Readme is not following the correct template. Please check out https://github.com/microverseinc/readme-template for inspiration.

**Review by arslanbisharat — CHANGES_REQUESTED** [review]
# Changes Requested, 

Hi  @AKeeganDev , 

You haven't made the requested changes. 

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: README.md [hunk 2]
```diff
@@ -1,73 +1,52 @@
...
-# Getting Started with Create React App
-
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
-
-## Available Scripts
-
-In the project directory, you can run:
-
-### `npm start`
+# Math Magicians
```
Comment (mateo951) [inline README.md:1]: Addressed ✔

**Review by mateo951 — APPROVED** [review]
### Hi @AKeeganDev, 

Your project is complete!  There is nothing else to say other than... it's time to merge it 📝
### Congratulations! 🎉 


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**